website/docs/cloud-docs/agents/telemetry.mdx

```diff
@@ -4,40 +4,104 @@ page_title: Telemetry - Terraform Cloud Agents - Terraform Cloud and Terraform E
+
+Optionally, you can pass the `-otlp-cert-file` or `TFC_AGENT_OTLP_CERT_FILE`.
+The agent will use a certificate at the path supplied to encrypt the client
+connection to the OpenTelemetry collector. When omitted, client connections are
+not secured.
+
+## OpenTelemetry Collector
+
+The OpenTelemetry Collector has a pluggable architecture. This opens up the
+ecosystem and makes it possible to leverage a wide array of telemetry platforms
+and technologies. The OpenTelemetry project distributes the collector in two
+flavors: `core` and `contrib`. For most uses, the `contrib` distribution will
+be most useful, as it includes a wide array of integrations for specific
+telemetry platforms and technologies which are not available in the `core`
+distribution. The `contrib` distribution is available as a pre-packaged
+Docker container labeled
+[otel/opentelemetry-collector-contrib](https://hub.docker.com/r/otel/opentelemetry-collector-contrib).
+
+## Compatibility
+
+Each released version of the Terraform Cloud Agent ships with a specific version
+of the OpenTelemetry client libraries built in. As these libraries and
+implementations change, it is possible that certain versions of the agent will
+become incompatible with newer versions of the OpenTelemetry collector. To
+track version compatibility, the table below will be updated whenever these
+version constraints become known.
 
-An example of the metric names that the agent will emit is `tfc-agent.terraform.plan-json.generate.bytes`. Breaking down those sections:
+| Terraform Cloud Agent version | Compatible OpenTelemetry Collector version |
+| ----------------------------- | ------------------------------------------ |
+| All versions                  | <= 0.48.0                                  |
 
-* `tfc-agent`: Metric names will be namespaced with tfc-agent to distinguish them from other metrics your system may be emitting.
-* `terraform`: Metric names may have a prefix with a component name, when a component is applicable.
-* `plan-json`: This shows that the metric in question is about the step of the agent process where JSON representations of the Terraform plan and Terraform provider schema are generated and uploaded.
-* `generate`: Specifically, it is about the generation of the JSON artifacts.
-* `bytes`: When a metric requires a unit in order to be understood, an un-abbreviated unit will be the last component of the metric name.
+### Running the collector
 
-In addition, agent metrics will follow some conventions around unit types:
+By default, the `contrib` distribution accepts telemetry from a wide variety of
+sources, and dumps information about the metrics and traces collected to stdout.
+To start the collector with default settings, run the following command:
 
-* All timing metrics (other than runtime metrics) will be measured in milliseconds.
-* All data size metrics will be measured in bytes.
+```
+docker run -p 127.0.0.1:4317:4317/tcp otel/opentelemetry-collector-contrib:latest
```

Comment:
i notice `latest` being configured in the command. I suppose what this should be will get complicated as we have newer version of the TFC agent that support newer instrumentation libraries, but maybe we should:

* make this `0.42.0` for now
* Forego a configuration of that in the example command (Which would still default to latest)
* Configure it as X.X.X, and change the instruction to say to replace the X's with the appropriate collector version, using the chart within the compatibility section; something along those lines

Reply:
Oh great call-out, thank you. I like the third option best as it allows us to refer to the compatibility chart, and doesn't force us to update the example commands as things change.

Reply:
On second thought though, this makes it more difficult to just copy and go, especially because this is a multi-line command (per guidelines for code in docs). I think for now, I'll make this 0.42.0, so that it works. If we can get `latest` working soon, we can update back to `latest`.

Reply:
that sounds good, and 🤞 for the potential utilization of latest soon!